.travis.yml

```diff
@@ -11,8 +11,7 @@ branches:
 
 env:
```

Comment:
I think this whole `env:` section can be removed

Reply:
(Along with -P$IT_PROFILE above)

Reply:
Done